### PR TITLE
Configure celery executor

### PR DESCRIPTION
**Pull request by btylerburton**
Makes CeleryExecutor the default.

Related:
- https://github.com/GSA/data.gov/issues/4539

### CODE REVIEW

File: .profile [hunk 1]
```diff
@@ -31,12 +31,12 @@ export APP_NAME=$(echo $VCAP_APPLICATION | jq -r '.application_name')
 # export REDIS_PASSWORD=$(vcap_get_service redis .credentials.password)
 # export REDIS_PORT=$(vcap_get_service redis .credentials.port)
 
-# export AIRFLOW__CELERY__BROKER_URL="$(vcap_get_service redis .credentials.uri)/0"
-# export BROKER_URL=$AIRFLOW__CELERY__BROKER_URL
+# GET VCAP VALS FOR DB & ELASTICACHE REDIS
+POSTGRES_DB_URI="$(vcap_get_service db .credentials.uri)"
+REDIS_URI="$(vcap_get_service redis .credentials.uri)/0?ssl_cert_reqs=CERT_OPTIONAL"
```
Comment (jbrown-xentity) [inline .profile:36]: Was this the line, the `?ssl_cert_reqs=CERT_OPTIONAL` was required?

**Review by jbrown-xentity — DISMISSED** [review]
Looks good. Just to clarify, we do want to do some testing at scale to validate if the celery executer works better, correct? If we decide that the local executor works better, will we just reverse this PR?

File: .profile (hunk 1 above)
Comment (btylerburton) [reply .profile:36]: Yes! That and replacing `redis` for `rediss` on line 38. Very sweetly undocumented anywhere, thank you very much. 😵‍💫 

https://github.com/apache/airflow/issues/15486#issuecomment-827900665

Comment (btylerburton) [reply .profile:36]: Eventually, we will have to secure that down, but at least for now it tells us what the issue was.

**Comment by btylerburton** [comment]
@jbrown-xentity I'm of two minds regarding that. I initially included the LocalExecutor code for future benchmarking, but the Celery stuff just ripped through the daily harvest job tasks so quickly I ended up ripping it out. 

We can certainly extract the files back out of this PR, but do you think it makes sense to keep them so we can do some proper benchmarking first?

**Comment by jbrown-xentity** [comment]
> @jbrown-xentity I'm of two minds regarding that. I initially included the LocalExecutor code for future benchmarking, but the Celery stuff just ripped through the daily harvest job tasks so quickly I ended up ripping it out.
> 
> We can certainly extract the files back out of this PR, but do you think it makes sense to keep them so we can do some proper benchmarking first?

I don't have a good feel for what is most likely to be efficient; that being said we'll need to try both. Maybe try local first before this is merged, and then try celery after this is merged? For the record, the pro's and cons as I see them: celery will be more configurable/optimizable, but cost more (redis server) and require more fancy configuration management. If celery is significantly slower with a 1-1 test, I would lean towards just using the local executor; but if it's faster or the same I would lean towards celery.

File: .profile (hunk 1 above)
Comment (jbrown-xentity) [reply .profile:36]: I should have caught that, we've done that already before... https://github.com/GSA/catalog.data.gov/blob/main/.profile#L92

Comment (btylerburton) [reply .profile:36]: i assume it's a feature of elasticache, but what i find really frustrating is there's no clear warning about this -- that i can find -- anywhere...

**Comment by btylerburton** [comment]
I'll go ahead and undelete the docker-compose and manifests with localexecutor support and then we can get this merged and move on.

**Comment by btylerburton** [comment]
@jbrown-xentity ok clear to review when you're ready...

**Review by jbrown-xentity — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
 	cf scale airflow-test-webserver -i 1
+	cf scale airflow-test-worker -i 1
 
 scale-down: ## Scales down CF airflow test deploy in current CF space
 	cf scale airflow-test-webserver -i 0
 	cf scale airflow-test-scheduler -i 0
+	cf scale airflow-test-worker -i 0
+
+push:
+	cf push --vars-file vars.<space>.yml
```
Comment (nickumia) [inline Makefile:20]: Am I missing something or should `<space>` have been a variable somehow?

Comment (btylerburton) [reply Makefile:20]: you are correct 🥇

**Pull request merged**